constellations/clients/orionFakeDataFiller/filler.go

```diff
@@ -35,6 +36,139 @@ func runFiller(hostAddress string) {
 		"description": "Students should take this course if they aim to take the AP Calculus Exam"
 	}`)
 	sendPostRequest(hostAddress+"/api/programs/create", body)
+
+	body1 := strings.NewReader(`{
+				"programId": "ap_java",
+				"name": "AP Java",
+				"grade1": 10,
+				"grade2": 12,
+				"description": "Students should take this course if they aim to take the AP Java Exam"
+			}`)
+	sendPostRequest(hostAddress+"/api/programs/create", body1)
+
+	body2 := strings.NewReader(`{
+				"programId": "sat_math",
+				"name": "SAT Math",
+				"grade1": 8,
+				"grade2": 11,
+				"description": "Students should take the course if they aim to take the SAT Math Exam"
+			}`)
+	sendPostRequest(hostAddress+"/api/programs/create", body2)
+
+	// Create semesters
+	body3 := strings.NewReader(`{
+				"semesterId": "2020_fall",
+				"title": "Fall 2020"
+			}`)
+	sendPostRequest(hostAddress+"/api/semesters/create", body3)
+
+	body4 := strings.NewReader(`{
+				"semesterId": "2021_summer",
+				"title": "Summer 2021"
+			}`)
+	sendPostRequest(hostAddress+"/api/semesters/create", body4)
+
+	body5 := strings.NewReader(`{
+				"semesterId": "2021_winter",
+				"title": "Winter 2021"
+			}`)
+	sendPostRequest(hostAddress+"/api/semesters/create", body5)
+
+	// Create locations
+	body6 := strings.NewReader(`{
+				"locationId": "xkcd",
+				"street": "11300 Gainsborough Rd",
+				"city": "Potomac",
+				"state": "MD",
+				"zipcode": "20854"
+			}`)
+	sendPostRequest(hostAddress+"/api/locations/create", body6)
+
+	body7 := strings.NewReader(`{
+				"locationId": "house1",
+				"street": "123 Sesame St",
+				"city": "Rockville",
+				"state": "MD",
+				"zipcode": "20854"
+			}`)
+	sendPostRequest(hostAddress+"/api/locations/create", body7)
+
+	// Create achievements
+	body8 := strings.NewReader(`{
+				"year": 2020,
+				"message": "100% of students scored above a 1550 on SAT!"
+			}`)
+	sendPostRequest(hostAddress+"/api/achievements/create", body8)
+
+	body9 := strings.NewReader(`{
+				"year": 2020,
+				"message": "5 students scored an 800 on SAT Math!"
+			}`)
+	sendPostRequest(hostAddress+"/api/achievements/create", body9)
+
+	body10 := strings.NewReader(`{
+				"year": 2019,
+				"message": "10 students scored a 5 on AP Java!"
+			}`)
+	sendPostRequest(hostAddress+"/api/achievements/create", body10)
+
+	// Create announcements
+
+	body11 := strings.NewReader(`{
+								var now = time.Now().UTC()
+								"postedAt": now,
+								"author": "Author Name",
+								"message": "The Summer 2020 session of SAT Math will start on Tuesday, June 30.",
+								"onHomePage": true
+							}`)
```

Comment:
I see what you're trying to do.

You can try this: 
```
now := time.Now().UTC()
nowJson, _ := now.MarshalJSON()
bodyStr := fmt.Sprintf(`{
	"postedAt": %s,
	"author": "%s",
	"message": "%s",
	"onHomePage": true
}`, nowJson, "Author Name", "The Summer 2020 session of SAT Math")
body11 := strings.NewReader(bodyStr)
```

Reply:
The `%s` are replaced by arguments, specifically `nowJson`, the author name and the custom message.
Use this when you need to add a specific time or have specific arguments you want to insert.